WordPress/openverse
# Update banner to match new designs and add analytics notification

## Fixes



Fixes #823 by @dhruvkb

## Description



> **Note**
> This PR targets #2117 i.e. `banner-style-refresh` and not `main`.

This PR
- updates the `VNotificationBanner` component to support
  - 4 natures (info, warning, success, error)
  - 2 variants (regular, dark)
- adds 4 natures x 2 variants x 2 breakpoints = 16 stories for these
- updates existing banners to use these new designs
- adds a banner for the analytics
- adds VRT for these Storybook stories
- updates existing VRTs to pass with the additional banner
- adds new icon assets (These are not a 1:1 match because some icons are not in the WP DL, requesting @fcoveram to provide)

## Testing Instructions


You can directly refer to updated snapshots in the "Files changed" tab. Or you can do the following: 

0. Check out the PR and run the dev server.
1. Clear all cookies.
2. Visit `http://localhost:8443/hi/?referrer=creativecommons.org`
3. You should see all three banners in the dark variant.
   ![Openly Licensed Images, Audio and More | Openverse](https://github.com/WordPress/openverse/assets/16580576/9be17038-2436-476c-9e88-39c65ff45228)

---

1. Visit `http://localhost:8443/hi/about/?referrer=creativecommons.org`
2. You should all all three banners in the regular variant.
   ![About Openverse | Openverse](https://github.com/WordPress/openverse/assets/16580576/6fa169a9-e87f-4d16-b3df-f6a3315faf79)

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
